modules/userId/index.js

```diff
@@ -349,11 +352,16 @@ function getStoredValue(submodule, key = undefined) {
   const storedKey = key ? `${storage.name}_${key}` : storage.name;
   let storedValue;
   try {
-    if (storage.type === COOKIE) {
-      storedValue = mgr.getCookie(storedKey);
-    } else if (storage.type === LOCAL_STORAGE) {
-      storedValue = getValueFromLocalStorage(submodule, storedKey);
-    }
+    submodule.enabledStorageTypes.find(storageType => {
```

Comment:
If you used a `forEach` instead of `find` and deleted the `return !!storedValue` line, wouldn't the result be the same?